prefect_docker/worker.py

```diff
@@ -152,6 +153,14 @@ class DockerWorkerJobConfiguration(BaseJobConfiguration):
         ),
     )
 
+    extra_args: Optional[Dict[str,Any]] = Field(
+        default=None,
+        title="Extra args",
+        description=(
+            "Extra args for dockerpy containers.create"
+        ),
+    )
```

Comment:
There's an opportunity to make the overall base job template more flexible. Naming this field as such will help towards that goal.
```suggestion
    container_create_kwargs: Optional[Dict[str,Any]] = Field(
        default=None,
        title="Container Configuration",
        description=(
            "Configuration for containers created by workers. See the [`docker-py` documentation](https://docker-py.readthedocs.io/en/stable/containers.html) for accepted values."
        ),
    )
```